README.md

```diff
@@ -9,6 +9,25 @@ We plan to enable image data and time-series data.
 
 Find the documentation [here](https://automl.github.io/Auto-PyTorch/development)
 
+## Workflow
+
+The rough description of the workflow of Auto-Pytorch is drawn in the following figure.
+
+<img src="figs/apt_workflow.png" width="500">
+
+In the figure, **Data** is provided by user and
+**Portfolio** is a predefined pool of machine learning algorithms, e.g. LightGBM and support vector machine, to solve either
```

Comment:
Portfolio is a greedy portfolio built based on the paper `AutoPyTorch Tabular`. They use the approach, PoSH-Auto-Sklearn to utilize portfolios in a meta-feature free approach